### DISCUSSION TITLE
网页上下文菜单右键翻译或划词翻译功能

### DISCUSSION THREAD

**Discussion by ACKLITS** [immersive-translate/immersive-translate#597] [Ideas]
### 新功能 | New Feature

1.网页上选择一段需要翻译的文本，右键直接翻译，可以设置翻译喜好，右键直接翻译（例如中-英，中-法，法-英之类的），可以是单词，句子，文本，小篇幅的翻译。
2.划词翻译功能。（如果能显示所选划词的词性 n. 名词、v.动词...就更好了）

### 为什么您想要添加这些功能 | Why you want to add these features

1.沉浸式翻译插件目前主要做的是网页全文翻译、pdf翻译，这是大篇幅的翻译，有时候仅仅只需要小篇幅的翻译。
2.划词翻译相对于全篇翻译文来说让人记忆更深刻，更加方便。（目前有类似的功能，鼠标悬浮+快捷键的方式，这种只能翻译段落不够灵活，只能作为划词翻译的一种形式）
3.希望沉浸式翻译插件越来越好，不要过于臃肿。

**Comment by theowenyoung** [comment]
有没有试过 鼠标悬停翻译？ 面板里可以手动开启。

**Comment by ACKLITS** [comment]
> 有没有试过 鼠标悬停翻译？ 面板里可以手动开启。

用过，用过😉，我上面有提过，但是不够灵活。
